packages/paste-core/components/button/src/types.ts

```diff
@@ -4,9 +4,12 @@ export type ButtonVariants = 'primary' | 'secondary' | 'destructive' | 'destruct
 export type ButtonStates = 'disabled' | 'loading' | 'default';
 export type ButtonTabIndexes = 0 | -1;
 
-export interface ButtonProps {
+export interface ButtonProps extends React.ButtonHTMLAttributes<HTMLButtonElement> {
   id?: never;
   className?: never;
+  style?: never;
+  width?: never | null;
+  height?: never | null;
```

Comment:
Remove all these `never` props on the interface it does nothing for us.

If anything its more confusing as it’ll try to auto complete the attribute, and then tell you once it’s done that you can’t use it.

That’ll at least hopefully solve some of the typing issues with classname and style. Setting them to null shouldn’t clash with the html types, unless it expects `undefined`

Reply:
Looks like they do want `undefined `. That seems to still work if I try to pass anything to either `className` or `style`. I removed `width` and `height` since they're not valid for button anyway.